xml/security_ldap.xml

```diff
@@ -389,290 +387,57 @@ MAY (userPassword $ searchGuide $ seeAlso $ businessCategory <co xml:id="co.ldap
-     </step>
-    </substeps>
-   </step>
-   <step>
-    <para>
-     Click <guimenu>OK</guimenu> to apply your settings and leave the user
-     configuration.
-    </para>
-   </step>
-  </procedure>
-
-<!--
-  <figure id="fig.ldap.y2.adset">
-   <title>&yast;: Additional LDAP Settings</title>
-   <mediaobject>
-    <imageobject role="fo">
-     <imagedata fileref="ldap_y2_adset.png" width="75%" format="PNG"/>
-    </imageobject>
-    <imageobject role="html">
-     <imagedata fileref="ldap_y2_adset.png" width="75%" format="PNG"/>
-    </imageobject>
-   </mediaobject>
-  </figure>
-  -->
-
-  <para>
-   The initial input form of user administration offers <guimenu>LDAP
-   Options</guimenu>. This allows you to apply LDAP search filters to the
-   set of available users. Alternatively open the module for configuring
-   LDAP users and groups by selecting <guimenu>LDAP User and Group
-   Configuration</guimenu>.
-  </para>
- </sect1>
-
- <sect1 xml:id="sec.security.ldap.slapd" os="sles;osuse">
-  <title>Manually Configuring an LDAP Server</title>
-
-  <para>
-   <remark>taroth 2019-02-21: @wbrown: the content of this section needs to be
-   revised for sure - could you please let me know what to remove, replace, or
-   add?</remark>
-   To conveniently access the configuration back-end, you use SASL external
-   authentication. For example, the following <command>ldapsearch</command>
-   command executed as &rootuser; can show the complete
-   <command>slapd</command> configuration:
-  </para>
-
-<screen>&prompt.user;ldapsearch -Y external -H ldapi:/// -b cn=config</screen>
+    </varlistentry>
+  </variablelist>
 
+  <!--taroth 2019-02-25: the following note and para are 'leftovers' of the former
+   section, check with wbrown if we need to keep them-->
   <note>
-<!-- bnc#911409 -->
    <title>LDAP Server Is Part of the Authentication Server</title>
    <para>
+    <remark>taroth 2019-02-25: @wbrown: is this note still valid?</remark>
     Basic LDAP Server initialization and configuration can be done within
     the Authentication Server &yast; module. For more information, see
     <xref linkend="sec.security.auth.yast"/>.
    </para>
   </note>
-
-  <!-- <sect2 xml:id="sec.security.ldap.slapd.start"> -->
-  <!--  <title>Starting and Stopping the Servers</title> -->
-   <para>
-    When the LDAP server is fully configured and all desired entries have
-    been made according to the pattern described in
-    <xref linkend="sec.security.ldap.data"/>, start the LDAP server as
-    &rootuser; by entering <command>sudo systemctl start
-    slapd</command>. To stop the server manually, enter the command
-    <command>sudo systemctl stop slapd</command>. Query the status of
-    the running LDAP server with <command>sudo systemctl status
-    slapd</command>.
-   </para>
    <para><remark>taroth 2019-02-21: @wbrown: is the following still true with
     389-ds?</remark>
     Use the &yast; <guimenu>&ycc_runlevel;</guimenu>, described in
```

Comment:
I don't think so. dscreate will setup your systemd services for you, and yast if it calls dscreate will do the same. So this may not be necessary.

Reply:
OK, thanks, removed this para.